uportal-war/src/main/webapp/media/skins/respondr/common/bootstrap/variables.less

```diff
@@ -14,7 +14,7 @@
 @gray-light:             lighten(@gray-base, 46.7%); // #777
 @gray-lighter:           lighten(@gray-base, 93.5%); // #eee
 
-@brand-primary:         darken(#428bca, 6.5%); // #337ab7
+@brand-primary:         darken(#008bca, 6.5%); // #337ab7
```

Comment:
Avoid modifying `bootstrap/variables.less` this is a vendor file, not a uPortal specific file.
Instead override from `defaultSkin/less/variables.less`.

:notebook: **Note:** `bootstrap/variables.less` does not exist in the uPortal 5 source, this file is pulled from NPM.